js/src/common/components/LinkButton.js

```diff
@@ -13,15 +13,17 @@ import Button from './Button';
  *   the `active` prop will automatically be set to true.
  */
 export default class LinkButton extends Button {
-  static initProps(props) {
-    props.active = this.isActive(props);
-    props.config = props.config || m.route;
+  static initAttrs(attrs) {
+    super.initAttrs(attrs);
+
+    attrs.active = this.isActive(attrs);
   }
 
-  view() {
-    const vdom = super.view();
+  view(vnode) {
+    const vdom = super.view(vnode);
 
-    vdom.tag = 'a';
+    vdom.tag = m.route.Link;
+    vdom.attrs.active = String(vdom.attrs.active);
```

Comment:
I don't understand what effect this is supposed to have. I must be missing something completely, but there's nothing in the parent class (and neither was in old code) that does something to this attribute, as far as I can tell.

Reply:
Tbh, I don't recall making this change. @datitisev do you remember why we might have added this?

Reply:
Will hopefully be cleaned up as part of #2265.